Scripts/Core/VRView.cs

```diff
@@ -139,56 +146,63 @@ public static Coroutine StartCoroutine(IEnumerator routine)
             return null;
         }
 
-        public void OnEnable()
+        public static void CreateCameraRig(ref Camera camera, ref Transform cameraRig)
```

Comment:
This change is causing a compiler error due to a reference to m_Camera below (I think it's from the merge w/ dev). I can get around it by just commenting out the line.

Reply:
I'd also consider switching "m_Camera" to "viewerCamera"; the s_ActiveView's static m_Camera ref.

Reply:
I just pushed this quick fix. It warrants additional testing though.

Reply:
This should be `camera` as that is passed in to the function, not viewerCamera.